### ISSUE TITLE
Odd wording in Sentence case section in Stripe Review Requirements 

### ISSUE THREAD

**Issue by jdwilkin4** [stripe/stripe-apps#639]
## Your Problem
There is odd wording in the last sentence in the Do column of the "Sentence case" section of the [review guidelines](https://stripe.com/docs/stripe-apps/review-requirements).

Here is the sentence:
Example: Add bank account For example: “Create subscription”

I think it would be much easier to read if it were changed to this:
For example: “Add bank account ”

## Existing Documentation URLs (If applicable)
https://stripe.com/docs/stripe-apps/review-requirements

**Comment by mjohnsonpint-stripe** [comment]
I agree this doesn't make much sense to me either. Let's see if @katb-stripe can come up with something a bit clearer.
